content/blog/recipes-0-2-0/index.Rmd

```diff
@@ -0,0 +1,86 @@
+
+Here's a real example from [Episode 8 of _Sliced_](https://www.kaggle.com/c/sliced-s01e08-KJSEks) where a column of data from Spotify had the artist(s) of a song: 
+
+```{r}
+library(recipes)
+spotify <- 
+  tibble::tribble(
+    ~ artists,
+    "['Genesis']",
+    "['Billie Holiday', 'Teddy Wilson']",
+    "['Jimmy Barnes', 'INXS']"
+  )
+recipe(~ artists, data = spotify) %>% 
+  step_dummy_extract(artists, pattern = "(?<=')[^',]+(?=')") %>% 
+  prep() %>% 
+  bake(new_data = NULL) %>% 
+  names()
+```
+
+Note that this step produces an "other" column and has arguments similar to `step_other()`. 
+
+`step_percentile()`, previously seem in the developer documentation, can determine the empirical distribution of the variable using the training set, then convert any value to the percentile of this distribution. 
+
+Finally, a new filtering function (`step_filter_missing()`) can filter out columns that have two many missing values (for some definition of "too many").
+
+## Other notable new features
+
+`step_zv()` now has a `group` argument. This can be helpful for models such as naive Bayes or quadratic discriminant analysis where the predictors must have at least two unique values _within each class_. 
+
+All recipe steps now officially support empty selections to be more aligned with dplyr and other packages that use tidyselect. For example, if a previous step removed all of the columns needed for a later step, the recipe does not fail when it is estimated (with the exception of `step_mutate()`). The documentation in `?selections` has been updated with advice for writing selectors when filtering steps are used. 
+
+There are new `extract_parameter_set_dials()` and `extract_parameter_dials()` methods to extract parameter sets and single parameters from a recipe. Since this is related to tuning parameters, the tune package should be loaded before they are used.  
+
+
+## Acknowledgements
+
+We'd like to thank everyone that has contributed since the last release: [&#x0040;arneschillert](https://github.com/arneschillert), [&#x0040;batpigandme](https://github.com/batpigandme), [&#x0040;DavisVaughan](https://github.com/DavisVaughan), [&#x0040;fenguoerbian](https://github.com/fenguoerbian), [&#x0040;gaborcsardi](https://github.com/gaborcsardi), [&#x0040;hadley](https://github.com/hadley), [&#x0040;hfrick](https://github.com/hfrick), [&#x0040;jimhester](https://github.com/jimhester), [&#x0040;juliasilge](https://github.com/juliasilge), [&#x0040;lionel-](https://github.com/lionel-), [&#x0040;maelle](https://github.com/maelle), [&#x0040;PursuitOfDataScience](https://github.com/PursuitOfDataScience), [&#x0040;sysilviakim](https://github.com/sysilviakim), and [&#x0040;topepo](https://github.com/topepo)
```

Comment:
How am I not on this list? 😄  I think you were in the wrong repo

Reply:
fixed